_site/about.html

```diff
@@ -109,7 +109,7 @@ <h1 class="post-title">About me</h1>
     <script type="text/javascript">
 
       var _gaq = _gaq || [];
-      _gaq.push(['_setAccount', 'UA-17795190-1']);
+      _gaq.push(['_setAccount', '']);
```

Comment:
@schmierkov the Google Analytics ID has been blanked out because of your `.env` file, you should have the ID `UA-17795190-1` in it as `GOOGLE_ANALYTICS_ID`.